cdap-docs/admin-manual/source/troubleshooting.rst

```diff
@@ -2,13 +2,29 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2014 Cask Data, Inc.
 
-============================================
-Troubleshooting CDAP
-============================================
+.. _admin-troubleshooting:
```

Comment:
:+1:  not sure about the developer manual, but I agree there is no reason not to consolidate at least the other 2.